dotnet/src/Connectors/Connectors.HuggingFace/Core/HuggingFaceClient.cs

```diff
@@ -293,11 +293,18 @@ private Uri GetTextGenerationEndpoint(string modelId)
 
         string body = await this.SendRequestAndGetStringBodyAsync(httpRequestMessage, cancellationToken)
             .ConfigureAwait(false);
+        try
+        {
+            // Attempt to deserialize as TextEmbeddingResponseType1
+            var response = DeserializeResponse<TextEmbeddingResponse>(body);
+            return response.ToList();
+        }
+        catch (KernelException ex)
+        {
+            // Eit
+            throw ex;
```

Comment:
If we don't do anything with this exception, should we remove `try/catch` block?

Reply:
Removed the try-catch method as it not required now